src/sonic-config-engine/tests/sample-graph-resource-type.xml

```diff
@@ -180,11 +180,6 @@
           <AttachTo>fortyGigE0/4</AttachTo>
           <SubInterface/>
         </PortChannel>
-        <PortChannel>
-          <Name>PortChannel1001</Name>
-          <AttachTo>fortyGigE0/1;fortyGigE0/2</AttachTo>
```

Comment:
>fortyGigE0/1;fortyGigE0/2 [](http://example.com/codeflow?start=20&length=25)

@neethajohn Did you intend to attach this PortChannel to some existing ports?

Reply:
I had copied simple-sample-graph.xml and made changes for my scenario. This part was untouched.